2_base_and_sensor.ipynb

```diff
@@ -256,20 +457,64 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Capture sensor information\n",
+    "In the previous section, we code the movement of the robot using our prior knowledge about the environment. In this section, we use the sensor to acquire environmental information to move the robot.\n",
+    "\n",
+    "We can use the following sensors installed in HSR:\n",
+    "\n",
+    "- Laser scanner: Two-dimensional measurement of distance to obstacles\n",
+    "\n",
+    "- RGB-D camera: A camera that can measure color information (RGB) + depth information (Depth)\n",
+    "\n",
+    "- IMU: Measures acceleration, angular acceleration, and magnetic force\n",
+    "\n",
+    "- Encoder: Measures each joint angle of the robot"
```

Comment:
"- Encoders: Measures each joint angle of the robot"

I think that there is an encoder for each joint, so I made the word "Encoder" plural as "Encoders".